api/testdata/statusResponse.json

```diff
@@ -1 +1 @@
-{ "meta": { "code": 200, "status": "ok", "version":"0.5.0" }, "data": [] }
\ No newline at end of file
+{ "meta": { "code": 200, "status": "ok", "version":"0.5.1-dev" }, "data": [] }
```

Comment:
this is sort of a side thing, but can we have a doc or issue somewhere that states where all the different places that tell us where all the "versions" live. For example, I know that for the backend, we have a `VersionNumber` const that lives in lib, but I'm unsure where the same thing exists on the frontend. Is it just the `package.json` file?

I know you've also linked to info that describes the way you want to version things, before, but I'd be nice to have a mini style guide for that. Eg, before it's released we call it `-dev`

Happy to make a paper doc or issue if we think that's the best place to keep that knowledge.